src/SQLStore/TableBuilder/MySQLTableBuilder.php

```diff
@@ -143,7 +143,10 @@ protected function doUpdateTable( $tableName, array $attributes = null ) {
 		}
 	}
 
-	private function getCurrentFields( $tableName ) {
+	/**
+	 * @return string[]|string[][]
```

Comment:
This is wrong, but is strictly correct because `$row->Type` could be an array and `str_replace` can run on an array, hence why it thinks you can have 2D array.